app/controllers/api/messages_controller.rb

```diff
@@ -0,0 +1,28 @@
+class Api::MessagesController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @game = params[:game_id].nil? ? current_game : Game.find_by_id(params[:game_id])
+    @game.update
+
+    recipientId = message_params['recipient'].to_i
+    senderId = message_params['sender'].to_i
+
+    new_params = message_params.except('recipient', 'sender')
+    @message = Message.new(new_params)
+    @message.sender = Team.find(senderId)
```

Comment:
I'm still not convinced that this "exclude these properties and add them as things" is the right approach for this, but again, MVP.